File: src/BaseHmiTypes/Screens/Controls/HmiRadarChartCategory.cs
using BaseHmiTypes.Common;
using BaseHmiTypes.Screens.Base;

namespace BaseHmiTypes.Screens.Controls;

public sealed class HmiRadarChartCategory
{
    public int? Index { get; set; }

    public HmiMultilingualText? Name { get; set; }

    public HmiProperty<double>? Minimum { get; set; }

    public HmiProperty<double>? Maximum { get; set; }
}

File: typescript/base-hmi-types/src/screens/controls/HmiRadarChartCategory.ts
import { HmiMultilingualText } from "../../common/HmiMultilingualText.js";
import { HmiProperty } from "../base/HmiProperty.js";

export class HmiRadarChartCategory {
  index?: number;
  name?: HmiMultilingualText;
  minimum?: HmiProperty<number>;
  maximum?: HmiProperty<number>;
}
